obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -562,6 +562,21 @@ std::vector<char> registerProcess(void)
 	return buffer;
 }
 
+std::vector<char> registerMemoryDump(void)
+{
+	std::vector<char> buffer;
+	uint8_t action     = 2;
```

Comment:
Should we but this in a constant / enum, why did we use the value 2?

Reply:
before that we use only 0 and 1 as register and unregister commands. 
yes using enum make sense now even more. will update .

Reply:
done